src-graphql/client.js

```diff
@@ -85,4 +85,115 @@ export default class Client {
       return response.model.node;
     });
   }
+
+  fetchQueryProducts(queryObject = {}, query = productConnectionQuery()) {
+    const queryArgStrings = [];
+    const options = {};
+
+    if (queryObject.title) {
+      queryArgStrings.push(`title:'${queryObject.title}'`);
+    }
+    if (queryObject.updatedAtMin) {
+      queryArgStrings.push(`updated_at:>='${queryObject.updatedAtMin}'`);
+    }
+    if (queryObject.createdAtMin) {
+      queryArgStrings.push(`created_at:>='${queryObject.createdAtMin}'`);
+    }
+    if (queryObject.productType) {
+      queryArgStrings.push(`product_type:'${queryObject.productType}'`);
+    }
+    if (queryObject.limit) {
+      options.first = queryObject.limit;
+    }
+    if (queryObject.sortBy) {
+      let sortKey;
+
+      switch (queryObject.sortBy) {
+        case 'title-ascending':
```

Comment:
Wondering if we should consider changing how consumers specify sort keys and order. Right now it's a little strange to have a single string that contains both concepts.

Thinking of something like:

```js
client.fetchQueryProducts({sortBy: 'updatedAt', sortDirection: 'desc'})
```

This matches up with the key more directly.

@minasmart thoughts? If we want to change it, I'm indifferent to making it part of this PR or in a future one.